examples/all-clusters-app/ameba/main/chipinterface.cpp

```diff
@@ -97,12 +97,9 @@ AmebaOTAImageProcessor gImageProcessor;
 #endif
 
 #if CONFIG_ENABLE_OTA_REQUESTOR
-extern "C" void amebaQueryImageCmdHandler(uint32_t nodeId, uint32_t fabricId)
```

Comment:
@pankore I couldn't see where `amebaQueryImageCmdHandler` is being called so I removed the params to them since they aren't used. Let me know if you see an issue with this.

Reply:
For posterity, @pankore and I spoke offline and `amebaQueryImageCmdHandler` is called from the ameba SDK. The ameba SDK will adapt to this change once this PR merges.